### PR TITLE
feat[w1.1]:adding feedback controller,adapter and dto

### PR DESCRIPTION
**Pull request by jelenabudisa**
Tested with database

### CODE REVIEW

**Comment by jelenabudisa** [comment]
[(5) W1.1 : As a patient I want to give feedback so that I could share my experience.](https://trello.com/c/k9WBa322/1-5-w11-as-a-patient-i-want-to-give-feedback-so-that-i-could-share-my-experience)

**Review by anciz — DISMISSED** [review]
Everything seems to be well done!

File: HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    [ApiController]
+    public class FeedbackController : ControllerBase
+    {
+        private readonly IFeedbackService _feedbackService;
+        public FeedbackController(IFeedbackService feedbackService)
+        {
+            _feedbackService = feedbackService;
+        }
+
+        [HttpPost("addFeedback")]
```
Comment (milana-todorovic) [inline HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs]: This isn't how [REST resources are usually named](https://restfulapi.net/resource-naming/), so I'd switch to a more conventional naming scheme when the rest of the methods are added.

**Review by milana-todorovic — DISMISSED** [review]
Seems ok other than the resource naming, but since the controller will definitely have more methods I'd fix it then.

**Review by milana-todorovic — DISMISSED** [review]

**Review by anciz — DISMISSED** [review]

**Review by anciz — APPROVED** [review]

File: HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs [hunk 2]
```diff
@@ -21,11 +21,19 @@ public FeedbackController(IFeedbackService feedbackService)
...
+       [HttpPost]
+        public ActionResult AddFeedback(FeedbackDTO feedbackDTO)
         {
-            Feedback feedback = FeedbackAdapter.FeedbackDTOToFeedback(feedbackDTO);
-            _feedbackService.AddFeedback(feedback);
+            try
+            {
+                Feedback feedback = FeedbackAdapter.FeedbackDTOToFeedback(feedbackDTO);
+                _feedbackService.AddFeedback(feedback);
+                return Ok();
```
Comment (milana-todorovic) [inline HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs:31]: 200 OK is alright here, although 201 Created with the Location header set might be better for post? (I'm not asking to change this btw, just saying that your team should discuss which HTTP codes you'll be using and when exactly if you haven't yet.)

**Review by milana-todorovic — APPROVED** [review]
Seems alright.

**Pull request merged**